specification/batch/resource-manager/Microsoft.Batch/stable/2022-01-01/BatchManagement.json

```diff
@@ -0,0 +1,6019 @@
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/CertificateProperties",
+          "description": "The properties associated with the certificate."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/ProxyResource"
+        }
+      ],
+      "description": "Contains information about a certificate."
+    },
+    "CertificateCreateOrUpdateParameters": {
+      "type": "object",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/CertificateCreateOrUpdateProperties",
+          "description": "The properties associated with the certificate."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/ProxyResource"
+        }
+      ],
+      "description": "Contains information about a certificate."
+    },
+    "ListCertificatesResult": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Certificate"
+          },
+          "description": "The collection of returned certificates."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The continuation token."
+        }
+      },
+      "description": "Values returned by the List operation."
+    },
+    "DeleteCertificateError": {
+      "type": "object",
+      "properties": {
+        "code": {
+          "type": "string",
+          "description": "An identifier for the error. Codes are invariant and are intended to be consumed programmatically."
+        },
+        "message": {
+          "type": "string",
+          "description": "A message describing the error, intended to be suitable for display in a user interface."
+        },
+        "target": {
+          "type": "string",
+          "description": "The target of the particular error. For example, the name of the property in error."
+        },
+        "details": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DeleteCertificateError"
+          },
+          "description": "A list of additional details about the error."
+        }
+      },
+      "required": [
+        "code",
+        "message"
+      ],
+      "description": "An error response from the Batch service."
+    },
+    "DetectorListResult": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DetectorResponse"
+          },
+          "description": "The collection of Batch account detectors returned by the listing operation."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The URL to get the next set of results."
+        }
+      },
+      "description": "Values returned by the List operation."
+    },
+    "DetectorResponse": {
```

Comment:
how are detectors created?  They only have a GET right now

Reply:
Through AppLens's [on-boarding process](https://applensdiagnostics.azurewebsites.net/?p=76). I don't believe users will create them. They are meant to be consumed through the portal.